mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -188,65 +276,121 @@ export class CompareRunView extends Component {
                       ? Utils.formatTimestamp(run.getStartTime())
                       : '(unknown)';
                     return (
-                      <td className='meta-info' key={run.run_uuid}>
-                        {startTime}
+                      <td className='data-value' key={run.run_uuid} style={colWidthStyle}>
+                        <Tooltip
+                          title={startTime}
+                          color='blue'
+                          overlayStyle={{ maxWidth: '400px' }}
+                        >
+                          {startTime}
+                        </Tooltip>
                       </td>
                     );
                   })}
                 </tr>
+              </tbody>
+              <tbody className='table-block' style={tableBlockWidthStyle}>
                 <tr>
                   <th
                     scope='rowgroup'
-                    className='inter-title'
+                    className='inter-title sticky_header'
                     colSpan={this.props.runInfos.length + 1}
+                    style={tableBlockWidthStyle}
                   >
-                    <h2>
-                      <FormattedMessage
-                        defaultMessage='Parameters'
-                        // eslint-disable-next-line max-len
-                        description='Row group title for parameters of runs on the experiment compare runs page'
+                    <CollapsibleSection
```

Comment:
Why do we only wrap the Switch component with `CollapsibleSection`?

Reply:
Then we have to split the whole table to be 3 tables (run-info table, param table, metric table), now they are in a table.

Reply:
Shall we split ? feel like kind of big change but I can do that.

Reply:
I think that's simpler.